### PR TITLE
Implementation of major refucktoring as we discussed on Siberalem

### PR DESCRIPTION
**Pull request by hdogan**
- [x] Rename project
- [x] Refucktor code base

### CODE REVIEW

**Comment by ahmet** [comment]
I think instead of removing one of them it'd be better we combine both and let anyone to choose her/his flavor, wdyt? :)

File: main.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+$response['headers'] = $headers;
+$response['method']  = $_SERVER['REQUEST_METHOD'];
+
+$body = file_get_contents('php://input');
+
+$response['body']  = (isset($_SERVER['CONTENT_TYPE']) && $_SERVER['CONTENT_TYPE'] === 'application/json') ? json_decode($body) : $body;
+$response['query'] = $_GET;
+
+header('Content-Type: application/json');
+echo json_encode($response);
```
Comment (umutphp) [inline main.php:26]: A contribution, small for me but big for code quality :)

> All PHP files MUST end with a single blank line. 
> [PHP FIG](https://www.php-fig.org/psr/psr-12/#22-files)

**Review by umutphp — CHANGES_REQUESTED** [review]

File: main.php (hunk 1 above)
Comment (hdogan) [reply main.php:26]: Fock PHP FIG and their nonsense *MUST* recommendations. But I'll make that change 😸

**Comment by hdogan** [comment]
> I think instead of removing one of them it'd be better we combine both and let anyone to choose her/his flavor, wdyt? :)

Shall they live together in same branch/repository?

I hate golang<sup>1</sup>, golang fan boys stop mentioning "gopher" word, it's a protocol <sup>[2]</sup>, it doesn't belong to golang 😡 

<sup>1</sup> Hating golang might be supposed to be a sarcasm. :trollface: 
<sup>2</sup> https://tools.ietf.org/html/rfc1436

**Comment by ahmet** [comment]
> Shall they live together in same branch/repository?

Doesn't matter as I'm the only one using this repo. The alternative may be you creating a dibek-php repo and we backlink each other in the readme 🤓

> golang fan boys stop mentioning "gopher" word, it's a protocol [2], it doesn't belong to golang 😡

Man, you're reminding me that I'm old 🥲

**Pull request merged**